### PR TITLE
Expanded definition of HealthStatus #244

### PR DESCRIPTION
**Pull request by j0rt3g4**
Expanded definition of HealthStatus #244

### CODE REVIEW

File: docset/windows/storage/get-physicaldisk.md [hunk 1]
```diff
@@ -193,14 +193,16 @@ Accept wildcard characters: False
...
-Specifies the health status of physical disks to get.
+Specifies the **Health Status** of physical disks to get.
 The acceptable values for this parameter are:
 
 - Healthy 
 - Unhealthy 
 - Unknown 
 - Warning
 
+Refers to the SMART counters of all Hard drives has, and represents a wait to get some failure prediction so you can generate de actions to change or repair the disk.
```
Comment (nenonix) [inline docset/windows/storage/get-physicaldisk.md]: Suggestion:
**Refers to the SMART counters present in all hard drives. This information can be used to take action to, for example, replace or repair a disk.**

Comment (nenonix) [reply docset/windows/storage/get-physicaldisk.md]: Alternate suggestion:
**Refers to the SMART counters of all hard drives, generating a failure prediction that can help you take action to change or repair the disk.**

**Comment by AndreaBarr** [comment]
@j0rt3g4  please fix the conflicts so that I can get this approved for merging.

**Comment by j0rt3g4** [comment]
@AndreaBarr  is there anything else pending on this one?

**Comment by AndreaBarr** [comment]
I'm still waiting for the statuses to pass to be able to merge it.

**Comment by j0rt3g4** [comment]
@AndreaBarr  is that something else I need to do?

**Comment by AndreaBarr** [comment]
@j0rt3g4 It might be that there are still conflicts on it but it's just not showing on my end. Would you please double check?

**Comment by j0rt3g4** [comment]
ok @AndreaBarr  will do

**Comment by j0rt3g4** [comment]
I will be doing another PR since this one is on a Source Repository that I have to re-create (and re-fork to work with).

**Comment by e0i** [comment]
This PR has been replaced by #988

**Pull request closed without merging**